modules/rosa-hcp-aws-privatelink-create-cluster.adoc

```diff
@@ -0,0 +1,93 @@
+//
+// * rosa_hcp/rosa-hcp-aws-privatelink-creating-cluster.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-aws-privatelink-create-cluster_{context}"]
+= Creating an AWS PrivateLink cluster
+
+You can create a private cluster on {hcp-title} using the {product-title} (ROSA) CLI, `rosa`.
+
+[NOTE]
+====
+AWS PrivateLink is supported on existing VPCs only.
+====
+
+.Prerequisites
+
+* You have available AWS service quotas.
+* You have enabled the ROSA service in the AWS Console.
+* You have installed and configured the latest {product-title} (ROSA) CLI, `rosa`, on your installation host.
+* You have created a VPC with at least one private subnet.
+
+.Procedure
+
+Creating a cluster with {hcp} can take around 10 minutes.
+
+. You can create a {hcp-title} cluster with multiple availability zones (Multi-AZ). Your machine's classless inter-domain routing (CIDR) must match your virtual private cloud's CIDR. See https://docs.openshift.com/container-platform/4.14/installing/installing_aws/installing-aws-vpc.html#installation-custom-aws-vpc-requirements_installing-aws-vpc[Requirements for using your own VPC] and link:https://docs.openshift.com/container-platform/4.14/installing/installing_aws/installing-aws-vpc.html#installation-custom-aws-vpc-validation_installing-aws-vpc[VPC Validation] for more information.
++
+[IMPORTANT]
+====
+If you use a firewall, you must configure it so that {product-title} can access the sites that it requires to function.
+
+For more information, see the AWS PrivateLink firewall prerequisites section.
+====
+
+. Create the account-wide IAM roles by running the following command:
++
+[source,terminal]
+----
+$ rosa create account-roles --hosted-cp
+----
+
+. Create the OIDC configuration by running the following command:
++
+[source,terminal]
+----
+$ rosa create oidc-config --mode=auto --yes
+----
++
+Save the OIDC configuration ID since you need it to create the operator roles.
++
+.Example output
+[source,terminal]
+----
+I: Setting up managed OIDC configuration
+I: To create Operator Roles for this OIDC Configuration, run the following command and remember to replace <user-defined> with a prefix of your choice:
+	rosa create operator-roles --prefix <user-defined> --oidc-config-id 28s4avcdt2l318r1jbk3ifmimkurk384
+If you are going to create a Hosted Control Plane cluster please include '--hosted-cp'
+I: Creating OIDC provider using 'arn:aws:iam::46545644412:user/user'
+I: Created OIDC provider with ARN 'arn:aws:iam::46545644412:oidc-provider/oidc.op1.openshiftapps.com/28s4avcdt2l318r1jbk3ifmimkurk384'
+----
+
+. Create the Operator roles by running the following command:
++
+[source,terminal]
+----
+$ rosa create operator-roles --hosted-cp --prefix <operator_roles_prefix> --oidc-config-id <oidc_config_id> --installer-role-arn arn:aws:iam::$<account_roles_prefix>:role/$<account_roles_prefix>-HCP-ROSA-Installer-Role
+----
+
+. You can create a private {hcp-title} cluster by running the following command:
```

Comment:
Same ISG guidance about using the imperative mood:
```suggestion
. Create a private {hcp-title} cluster by running the following command:
```

If this is meant to be optional, I'd suggest writing it the same way but with "Optional:" at the beginning:
```suggestion
. Optional: Create a private {hcp-title} cluster by running the following command:
```